esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -3338,6 +3339,12 @@ void getSummaryStatsByQueryId(const char *target, const char *queryId, const cha
         DBGLOG("getSummaryStatsByQueryId(): '%s' => '%s'", control.str(), sb.str());
     }
 
+    if (returnAllRawStats)
+    {
+        toXML(queryAggregates, rawStats);
```

Comment:
Should we also support JSON? 
toJSON(queryAggregates, rawStats);

Reply:
JSON is my preferred defaults - but I was expecting the same result "shape", just with more items of different time periods?

Reply:
@GordonSmith  I can check and return a JSON format. A time period is specified in the WUQueryGetSummaryStats request. Are you saying that I should return the Stats without using the time period?